src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ShopUser.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource class="%sylius.model.shop_user.class%" shortName="ShopUser">
+        <attribute name="route_prefix">shop</attribute>
+
+        <collectionOperations />
+
+        <itemOperations>
+            <itemOperation name="shop_get">
```

Comment:
If this endpoint is needed, is it well secured so that only logged in user has access to their own information?